modules/tinymce/src/core/main/ts/api/dom/TreeWalker.ts

```diff
@@ -96,20 +96,20 @@ class DomTreeWalker {
     }
   }
 
-  private findPreviousNode(node: Node, startName: 'lastChild', siblingName: 'previousSibling', shallow?: boolean) {
+  private findPreviousNode(node: Node | undefined, shallow?: boolean): undefined | Node {
```

Comment:
Bit of a nice cleanup I think I just copied the function so that is why it had params. They are clearly not needed.